File: app/src/main/java/com/neome/java/api/home/main/msg/MsgReaction.java
// neome.ai API - do not change
//

package com.neome.java.api.home.main.msg;

import com.neome.java.api.core.base.dto.DtoChatMessageOffset;

import org.jetbrains.annotations.Nullable;

@SuppressWarnings("unused")
public class MsgReaction extends MsgOffset
{
  @Nullable
  public DtoChatMessageOffset chatMessageOffset;

  @Nullable
  public String reaction;
}

File: app/src/main/java/com/neome/api/home/main/msg/MsgReaction.kt
// neome.ai API - do not change
//
@file:Suppress("unused", "MemberVisibilityCanBePrivate", "CanBeParameter", "PropertyName")


package com.neome.api.home.main.msg

import com.neome.api.core.base.dto.DtoChatMessageOffset
import com.neome.api.home.main.msg.MsgOffset

interface MsgReaction : MsgOffset
{
  val chatMessageOffset: DtoChatMessageOffset?
  val reaction: String?
}